indexer/config.yml

```diff
@@ -1,18 +1,22 @@
 source:
-  type: oura
-  network: mainnet
-  socket: relays-new.cardano-mainnet.iohk.io:3001
-  bearer: Tcp
+  type: cardano_net
```

Comment:
let's add a folder called `configs` and make a couple of examples like 
`oura_based_config`, `cardano_net_based_config`